cmd/fleetctl/package.go

```diff
@@ -377,13 +377,31 @@ func packageCommand() *cli.Command {
 			}
 
 			path, _ = filepath.Abs(path)
+			pathBase := filepath.Base(path)
+			var installInstructions = "double-click the installer"
+			var deviceType string
+			switch c.String("type") {
+			case "pkg":
+				installInstructions += fmt.Sprintf(" or run the command `installer -pkg \"%s\" -target /`", pathBase)
```

Comment:
Don't you need `sudo` before `installer`?

Reply:
You're right, I missed that. Thanks 🙂